packages/base-styles/_colors.native.scss

```diff
@@ -102,6 +102,7 @@ $dark-ultra-dim: #ffffff14; //rgba(255, 255, 255, 0.08);
 $app-background: $white;
 $app-background-dark: $black;
 $app-background-dark-alt: $background-dark-elevated;
+$app-safe-area-background: #141414;
```

Comment:
Since this is only applied in dark mode, I'm wondering if we could rename it to `app-safe-area-background-dark`. WDYT?

https://github.com/WordPress/gutenberg/blob/98eb51c1f6c582e21da482277bdfd354c871a227/packages/edit-post/src/components/header/header-toolbar/style.native.scss#L10-L11

https://github.com/WordPress/gutenberg/blob/98eb51c1f6c582e21da482277bdfd354c871a227/packages/edit-post/src/components/layout/style.native.scss#L8-L9

Reply:
Nice catch! Updated in https://github.com/WordPress/gutenberg/pull/52336/commits/414ddc8483a367c0e5cc5548bf30c4cb4467798f thanks!